pynance/dash_viz/plot_flow_test.py

```diff
@@ -0,0 +1,90 @@
+
+from __future__ import print_function, absolute_import
+
+import unittest
+
+import pandas as pd
+
+from .plot_flow import app, csvtype_string2description, update_output,\
+    onselect_csvtype, update_csvtype_store, make_cashflow_figure
+from ..textimporter import SupportedCsvTypes
+
+# see https://github.com/plotly/dash/issues/297
+
+
+class DashTestCase(unittest.TestCase):
+    def test_build_app(self):
+        self.assertFalse(app is None)
+
+    def test_csvtype_string2description(self):
+        self.assertEqual(SupportedCsvTypes.DKBCash,
+                         csvtype_string2description('DKBCash'))
+        self.assertEqual(SupportedCsvTypes.DKBVisa,
+                         csvtype_string2description('DKBVisa'))
+
+    def test_update_output_None(self):
+        self.assertFalse(update_output(None, "") is None)
+
+    def test_onselect_csvtype(self):
+        dropdown_values = [None, "DKBCash", "DKBVisa"]
+        onselect_response = [False, True, True]
+
+        for expected, selected in zip(onselect_response, dropdown_values):
+            result = onselect_csvtype(selected)
+            # TODO: how to test if propagation of value change

```

Comment:
Don't know either :/ Maybe we should make an issue for this?

Reply:
opened #36